src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -838,4 +814,10 @@
   <data name="ColorScheme_RenameErrorTip.Title" xml:space="preserve">
     <value>This color scheme name is already in use.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SettingContainer_OverrideIntro.Text" xml:space="preserve">
+    <value>Overrides a setting</value>

```

Comment:
```suggestion
    <value>This more-specific setting overrides a value from another profile.</value>
```

"Overrides a setting" is engineer-speak vagueness. Can we brainstorm something better?

You'll want to be careful. When we make this into a hyperlink, we are going to need to support the translator changing the location of the link (because it is the object of the sentence, or indirect object, or qualifier, or something (@miniksa will be able to tell me what part of speech "profile name" is in the sentence "This overrides a value in [profile name]").) within the sentence.

Reply:
> "Overrides a setting" is engineer-speak vagueness. Can we brainstorm something better?
Summoning @cinnamon-msft for word-smithing.
The spec says "Overrides Base Layer" and I think we still would want this to be a fairly short string.

> You'll want to be careful. When we make this into a hyperlink, we are going to need to support the translator changing the location of the link

Good call. I'll have to use `fmt` and make it something like `Overrides {}` (comment: "{}" is the object being overridden (possibly a sentence)). Then for now, I'll just replace `{}` with "a setting" (localized) until hyperlinks come in.

Reply:
Discussed offline with @cinnamon-msft. Going with:
```
This overrides the value in {}.
```
Temporarily replacing {} with "another layer" until the hyperlink functionality goes in.

Reply:
![demo](https://user-images.githubusercontent.com/11050425/106798503-e9a0b500-6612-11eb-830f-d7d2e3b54cc5.png)

Reply:
Make sure `another layer` is a separate string. Understand that if you want to make it an actual hyperlink, you will _need_ to parse the string at runtime to know where to break it to insert the link.

Reply:
It would be very fun to write a `fmt` adapter for `TextBlock` (!), but... not today

Reply:
![demo 2](https://user-images.githubusercontent.com/11050425/106799764-7bf58880-6614-11eb-91e6-a3bddcd216ce.png)

last minute change: "another layer" --> "a lower layer"